tests/scenarios/github_test_12/cmd3

```diff
@@ -0,0 +1,7 @@
+source: ubuntu-1-deployment
+cmd: wget --tries=1 142.250.193.46
+result: failed
+---
+operation: Network
+condition: SOCK_STREAM
+action: Block
```

Comment:
why "Block"?

Reply:
The policy action is allow but default posture is Block, this will match the default Posture.